astro/src/content/docs/release-notes/index.mdx

```diff
@@ -122,7 +122,23 @@ Looking for release notes older than 1.23.0? Look in the [release notes archive]
 <ReleaseNoteHeading version='1.52.0' releaseDate='August 8, 2024' name="Passkey Platypus" />
 
 <GeneralMigrationWarning>
+  **User Registrations API**
+
   When using the User Registrations API, the `data` field for the FusionAuth application with Id `3c219e58-ed0e-4b18-ad48-f4f92793ae32` may now contain a `preferences` object. This object is reserved and should not be modified.
+
+  **Upgrading in an air-gapped configuration**
+
+  If you are not using an air-gapped license, this message can be disregarded. Have a good day!
+
+  For those running in an air-gapped configuration, you'll want to review this note. To ensure your premium features remain active after upgrading, please do the following:
+
+  * Navigate to the [Plan page](https://account.fusionauth.io/account/plan) in your FusionAuth account
+  * Pick up your license key and newly generated license text
```

Comment:
Do you need to regenerate the air gapped license text? Or is whatever is there good to go?

Reply:
Whatever is there is good to go. It automatically gets regenerated when they visit they page.